From 104005b73efb741d3396c75f348eb9f93a1dbf4a Mon Sep 17 00:00:00 2001
From: Johnny Mnemonic <jm@machine-hall.org>
Date: Thu, 5 Dec 2024 21:32:16 +0100
Subject: [PATCH 18/39] Replay "Implement C23 cospi" also for ia64

See 0ae0af68d8fa3bf6cbe1e4f1de5929ff71de67b3 for reference.
---
 sysdeps/unix/sysv/linux/ia64/libm.abilist | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/sysdeps/unix/sysv/linux/ia64/libm.abilist b/sysdeps/unix/sysv/linux/ia64/libm.abilist
index 30779aa3dd..cf84a395e9 100644
--- a/sysdeps/unix/sysv/linux/ia64/libm.abilist
+++ b/sysdeps/unix/sysv/linux/ia64/libm.abilist
@@ -1156,3 +1156,11 @@ GLIBC_2.40 logp1f32x F
 GLIBC_2.40 logp1f64 F
 GLIBC_2.40 logp1f64x F
 GLIBC_2.40 logp1l F
+GLIBC_2.41 cospi F
+GLIBC_2.41 cospif F
+GLIBC_2.41 cospif128 F
+GLIBC_2.41 cospif32 F
+GLIBC_2.41 cospif32x F
+GLIBC_2.41 cospif64 F
+GLIBC_2.41 cospif64x F
+GLIBC_2.41 cospil F
-- 
2.51.0

